projects/1.18/assets/enderite-lieonlion/lolenderite/lang/zh_cn.json

```
@@ -0,0 +1,35 @@
+{
+	"item.lolenderite.enderite_ingot": "末影合金锭",
+	"item.lolenderite.raw_enderite": "原生末影合金",
+	"item.lolenderite.enderite_fragment": "末影合金碎片",
+	"item.lolenderite.enderite_helmet": "末影合金头盔",
+	"item.lolenderite.enderite_chestplate": "末影合金胸甲",
+	"item.lolenderite.enderite_leggings": "末影合金护腿",
+	"item.lolenderite.enderite_boots": "末影合金靴子",
+	"item.lolenderite.enderite_sword": "末影合金剑",
+	"item.lolenderite.enderite_shovel": "末影合金锹",
+	"item.lolenderite.enderite_pickaxe": "末影合金镐",
+	"item.lolenderite.enderite_axe": "末影合金斧",
+	"item.lolenderite.enderite_hoe": "末影合金锄",
+	"block.lolenderite.enderite_block": "末影合金块",
+	"block.lolenderite.enderite_ore": "末影合金矿石",
+	"item.lolenderite.obsidian_infused_enderite_ingot": "黑曜石灌注末影合金锭",
+	"item.lolenderite.obsidian_infused_enderite_sword": "黑曜石灌注末影合金锭剑",
+	"item.lolenderite.obsidian_infused_enderite_shovel": "黑曜石灌注末影合金锭锹",
+	"item.lolenderite.obsidian_infused_enderite_pickaxe": "黑曜石灌注末影合金锭镐",
+	"item.lolenderite.obsidian_infused_enderite_axe": "黑曜石灌注末影合金锭斧",
+	"item.lolenderite.obsidian_infused_enderite_hoe": "黑曜石灌注末影合金锭锄",
+	
+	"advancement.lolenderite.again": "又来？",
+	"advancement.lolenderite.again.desc": "在锄头上浪费更多的末影合金，并对一切感到后悔",
+	"advancement.lolenderite.cant_stop_me_now": "现在无法阻止我",

```

| **审查级别** |
|--------------|
| ℹ️ 信息 / 不阻止合并  |

[The Score - Can't Stop Me Now (Lyric Video) - YouTube](https://www.youtube.com/watch?v=ZRRt8EyYNvg)

```suggestion
	"advancement.lolenderite.cant_stop_me_now": "现在无法阻止我",
```
原文：**Can't Stop Me Now**